net/lora/node/src/mac/LoRaMac.c

```diff
@@ -1973,8 +1981,8 @@ ScheduleTx(void)
 
             // Send later - prepare timer
             LoRaMacState |= LORAMAC_TX_DELAYED;
-            hal_timer_stop(&TxDelayedTimer);
-            hal_timer_start(&TxDelayedTimer, duty_cycle_time_off);
+            os_cputime_timer_stop(&TxDelayedTimer);
+            os_cputime_timer_relative(&TxDelayedTimer, duty_cycle_time_off);
```

Comment:
I guess duty_cycle_time_off is in microseconds here. I would double check that to be sure.

Reply:
It is in microseconds. As you can see from the original code, TxDelayedTimer is initialised using `LORA_MAC_TIMER_NUM`, which is configured at 1MHz:

https://github.com/apache/mynewt-core/blob/10fd2c3c2358accf9bba5a569e133da5ec7b24c1/net/lora/node/src/mac/LoRaMac.c#L1977